adapters/tlsio_wolfssl.c

```diff
@@ -999,6 +1010,12 @@ int tlsio_wolfssl_setoption(CONCRETE_IO_HANDLE tls_io, const char* optionName, c
             tls_io_instance->ignore_host_name_check = *server_name_check;
             result = 0;
         }
+        else if (strcmp("debugLog", optionName) == 0)
+        {
+            (void)wolfSSL_Debugging_ON();
+            (void)wolfSSL_SetLoggingCb(&logging_callback);
+            result = 0;
```

Comment:
how about doing this in init and wrap around macro DEBUG_WOLFSSL ?

marco comes from wolfssl/options.h

Reply:
I thought about having it enabled when WolfSSL is built with the debug flag but it's very verbose. 

This way we are backward compatible even in cases where WolfSSL was built with debug but we can enable it using the option if we need to investigate more.